workflow_array_ephys/paths.py

```diff
@@ -4,12 +4,9 @@
 
 def get_ephys_root_data_dir():
     root_data_dirs = dj.config.get('custom', {}).get('ephys_root_data_dir', None)
-
-    return root_data_dirs
-
+    return root_data_dirs if root_data_dirs else None
```

Comment:
Would the `get` method in the above statement set `root_data_dirs` to `None` if the key doesn't exist?  In which case, this if-else statement is not necessary.

Reply:
That's a good idea. I've added this to the batch of changes.